src/Events/Calendar_Embeds/Controller.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Manages the External Calendar Embeds Feature.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Calendar_Embeds
+ */
+
+namespace TEC\Events\Calendar_Embeds;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+
+/**
+ * Class Controller
+ *
+ * @since TBD
+
+ * @package TEC\Events\Calendar_Embeds
+ */
+class Controller extends Controller_Contract {
+
+	/**
+	 * @inheritDoc
+	 */
+	public function do_register(): void {
+		$this->container->singleton( Calendar_Embeds::class );
+
+		$this->add_actions();
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function unregister(): void {
+		$this->remove_actions();
+	}
```

Comment:
I am against the extra code just for the sake of it. Your current code has an unregister which calls the real unregister... Why not just have the add/remove filters in do_register and unregister and avoid having real and fake methods ?

Reply:
Hmm. That seems to go against the way we use controllers elsewhere. Maybe we can discuss with @bordoni about that. I understand your reasoning, though.